samwise.js

```diff
@@ -1,9 +1,10 @@
-/**
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?